completions/bash/podman

```diff
@@ -1018,6 +1018,7 @@ _podman_network_create() {
 	    ;;
     esac
 }
+
 _podman_network_inspect() {
     local options_with_args="
      "
```

Comment:
Ca also fix `--format -f`  to `local options_with_args` for network_inpect in this PR?

Reply:
Fixed